src/Bicep.LangServer/Snippets/Templates/res-app-insights-auto-scale-settings.bicep

```diff
@@ -0,0 +1,62 @@
+﻿// Application Insights Auto Scale Settings
+resource ${1:appInsightsAutoScaleSettings} 'Microsoft.Insights/autoscalesettings@2015-04-01' = {
+  name: ${2:'name'}
+  location: resourceGroup().location
+  tags: {
+    Application_Type: '${3|web,other|}'
+    'hidden-link:${resourceGroup().id}/providers/Microsoft.Web/serverfarms/${4:appServicePlan}': 'Resource'
+  }
+  properties: {
+    name: ${5:'name'}
+    profiles: [
+      {
+        name: ${6:'name'}
+        capacity: {
+          minimum: ${7:'minimum'}
+          maximum: ${8:'maximum'}
+          default: ${9:'default'}
+        }
+        rules: [
+          {
+            metricTrigger: {
+              metricName: ${10:'name'}
+              metricResourceUri: '${resourceGroup().id}/providers/Microsoft.Web/serverfarms/${4:appServicePlan}'

```

Comment:
Again, use a symbolic ref to the app service:
```
metricResourceUri: appService.id
```

Reply:
Updated here:
https://github.com/Azure/bicep/commit/ea170a6cefb1cfd227d5ffb7f8242f63aa441032